ci-operator/step-registry/openshift-qe/cluster-density/openshift-qe-cluster-density-commands.sh

```diff
@@ -16,11 +16,16 @@ ES_USERNAME=$(cat "/secret/username")
 
 git clone https://github.com/cloud-bulldozer/e2e-benchmarking
 pushd e2e-benchmarking/workloads/kube-burner
```

Comment:
We could also consider moving to `kube-burner-ocp-wrapper` here.

Reply:
ack - I am happy to help move that repo to use `ocp-wrapper` since we store results differently w/ the `ocp-wrapper` -- making it easier to compare results.

Reply:
Awesome, think this can be a separate pr/effort. Definitely want to know more about how we can more easily compare results